diff --git a/Makefile b/Makefile
index 465a24c..6198fb1 100644
--- a/Makefile
+++ b/Makefile
@@ -404,7 +404,7 @@ $(SRCBINDIR)/podman$(BINSFX): $(SOURCES) go.mod go.sum | $(SRCBINDIR)
 
 $(SRCBINDIR)/podman-remote-static-linux_%: GOARCH = $(patsubst $(SRCBINDIR)/podman-remote-static-linux_%,%,$@)
 $(SRCBINDIR)/podman-remote-static-linux_%: GOOS = linux
-$(SRCBINDIR)/podman-remote-static $(SRCBINDIR)/podman-remote-static-linux_amd64 $(SRCBINDIR)/podman-remote-static-linux_arm64: $(SRCBINDIR) $(SOURCES) go.mod go.sum
+$(SRCBINDIR)/podman-remote-static $(SRCBINDIR)/podman-remote-static-linux_amd64 $(SRCBINDIR)/podman-remote-static-linux_arm64 $(SRCBINDIR)/podman-remote-static-linux_loong64: $(SRCBINDIR) $(SOURCES) go.mod go.sum
 	CGO_ENABLED=0 \
 	$(GO) build \
 		$(BUILDFLAGS) \
